libpod/oci.go

```diff
@@ -798,6 +801,14 @@ func (r *OCIRuntime) execContainer(c *Container, cmd, capAdd, env []string, tty
 		return nil, errors.Wrapf(err, "cannot start container %s", c.ID())
 	}
 
+	if preserveFDs > 0 {
```

Comment:
Is this safe to do here? We haven't started the command yet

Reply:
hm... don't we do that at line 800?

Reply:
Looks like we do - having trouble reading my own code this morning. Should be fine then.